docs/drupal/services/redis.md

```diff
@@ -6,7 +6,7 @@ We recommend using [Redis](https://redis.io/) for internal caching. Add the Redi
 {% tab title="docker-compose.yml" %}
 ```yaml
   redis:
-    image: amazeeio/redis
+    image: uselagoon/redis
```

Comment:
I'm not sure if we want to use the plain redis or if we should use a version in the example

Reply:
Yeah, let's pick a version - likely 5 for now